utils/firebaseNotification.js

```diff
@@ -0,0 +1,81 @@
+import messaging from '@react-native-firebase/messaging'
+import {
+  Alert
+} from 'react-native';
+export default class FirebaseNotification {
+
+  async checkPermission() {
+    const authStatus = await messaging().requestPermission();
+
+    const { AUTHORIZED, PROVISIONAL } = messaging.AuthorizationStatus;
+    const isAuthorized = authStatus === AUTHORIZED;
+    const isProvisional = authStatus === PROVISIONAL;
+
+    const enabled = isAuthorized || isProvisional;
+    // const enabled =
+    //   authStatus === messaging.AuthorizationStatus.AUTHORIZED ||
+    //   authStatus === messaging.AuthorizationStatus.PROVISIONAL;
```

Comment:
Veerji, can you remove this commented code as well? I don't think we need it in production code.

Reply:
Fixed Veer ji